lib/src/sep/0099/hotswap.dart

```diff
@@ -151,6 +149,31 @@ class Hotswap {
       _httpClient.close();
     }
   }
+
+  bool _checkIntegrity(
+    Transaction originalTransaction,
+    Transaction signedTransaction,
+  ) {
+    final originalOperations = originalTransaction.operations;
+    final signedOperations = signedTransaction.operations;
+
+    if (originalOperations.length != signedOperations.length) {
+      return false;
+    }
+
+    final originalXdrBase64s = Set<String>.from(
```

Comment:
now you loop twice you can loop onces with a for loop.

```dart
for(...) {
   final org = originalOperations[0].toXdrBase64();
   final signed = originalOperations[0].toXdrBase64();
   if (org != signed) {
     return false;
   }
}
```

Reply:
There are only a few operations. I think the order of the operations in the transaction can be changed as well, since we have no idea what the anchors do in their implementation.

Reply:
The order of operations should not change. If it changes it fails. We can be very strict here. The anchor should only sign the transaction so this should never happen.